### PR TITLE
refactor(labware-library): Update landing content and instructions copy

### LINKED ISSUE #3977: LC: Landing page and form instruction copy edits
## overview

As a labware creator user, I would like to know exactly how to use the labware creator and where to find help.

## Implementation Details
- Add blue opentrons button styles based on website style-guide
- Add button for [instructions/support link](https://support.opentrons.com/en/articles/3136504-creating-custom-labware-definitions)
- Update 'Landing' Page copy and links
- Update form instructions
- Update form labels/text/options where applicable

## Acceptance Criteria
- [ ] Copy changes match [edits doc](https://docs.google.com/document/d/1JGlbm4QqIrrCDvuC9bHvcpqoqD_aAUovAby15zAiHvY/edit#heading=h.wu19f5ftx31x)
- [ ] No typos or grammar issues

### PR DESCRIPTION
**Pull request by Kadee80**
## overview

closes #3977

## changelog

- refactor(labware-library): Update landing content and instructions copy

## review requests

http://sandbox.labware.opentrons.com/lc_copy-edits/create/

**Note:** Punted on the subscript for XY offset and spacing since this text is driven by data strings and there is no subscript S unicode character.

- [ ] Layout looks good
- [ ] Copy/changes match doc
- [ ] Code is sane

### CODE REVIEW

File: labware-library/src/labware-creator/components/IntroCopy.js [hunk 1]
```diff
@@ -37,10 +35,11 @@ const IntroCopy = () => (
     <div className={styles.callout}>
       <p>
         <strong>Please note:</strong> We strongly recommend you reference
-        mechanical drawings to ensure accurate measurements for defining
-        labware, only relying on manual measurements to supplement missing
-        information. To learn more about ways to access mechanical drawings from
-        manufacturers, please refer to this guide.
+        mechanical drawings directly from your supplier/manufacturer and only
+        rely on manual measurements (using callipers when possible) to
```
Comment (IanLondon) [inline labware-library/src/labware-creator/components/IntroCopy.js]: ~callipers~ calipers

File: labware-library/src/labware-creator/index.js [hunk 2]
```diff
@@ -373,7 +377,7 @@ const App = () => {
           setValues,
         }) => (
           <div className={styles.labware_creator}>
-            <h2>Custom Labware Creator</h2>
+            <h2>Custom Labware Creator BETA</h2>
```
Comment (IanLondon) [inline labware-library/src/labware-creator/index.js]: 😌

File: labware-library/src/labware-creator/index.js [hunk 3]
```diff
@@ -592,13 +596,32 @@ const App = () => {
...
                   <div className={styles.flex_row}>
                     <div className={styles.instructions_column}>
-                      <p>
-                        Reference the <strong>inside</strong> of the well.
-                        Ignore any lip.
-                      </p>
-                      <p>
-                        Diameter helps the robot locate the sides of the wells.
-                      </p>
+                      {values.labwareType === 'tubeRack' ||
```
Comment (IanLondon) [inline labware-library/src/labware-creator/index.js]: you can use the helper fn `displayAsTube(values)` here

File: labware-library/src/labware-creator/components/IntroCopy.js (hunk 1 above)
Comment (Kadee80) [reply labware-library/src/labware-creator/components/IntroCopy.js]: changing now

**Pull request merged**